.travis.yml

```diff
@@ -55,6 +55,15 @@ matrix:
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" COVERAGE="1"
     - os: linux
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" RELEASE="1"
+    # Linux-X86_64 (with GDB tests)
```

Comment:
Sounds good.

On Fri, Jun 10, 2016 at 12:26 PM, Francis Ricci <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/facebook/ds2/pull/356#discussion_r66668272>:
>
> > @@ -55,6 +55,15 @@ matrix:
> >        env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" COVERAGE="1"
> >      - os: linux
> >        env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" RELEASE="1"
> > +    # Linux-X86_64 (with GDB tests)
>
> Partially because I figured that they were likely to break on different
> changes, and this makes it easier to find the signal. And also partially
> because it makes the travis env line less long and complicated.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/356/files/55e7bcac97863805496b95cf1d3e957a31822006..891a9222fe2a3b18f4136e7173b8507bc7f70cb7#r66668272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbJfUxILS3gSLNWPlJUkWsnh3oRupks5qKbpsgaJpZM4InmxB>
> .
>



-- 
Stephane Sezer